packages/web-components/src/text/text.options.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * TextSize constants
+ * @public
+ */
+export const TextSize = {
```

Comment:
What is the purpose of this const object?
It is exported internally, but not by the component's index, not used anywhere besides the type bellow.
How is this better than just using a literal union directly in the type?

Reply:
Synced offline - I'll write something up.

Reply:
Exported these - that was a mistake.

In terms of the use here, the const object is a move we made to support improved strict typings for partners while still ensuring that general "strings" will work in JS land. For TS users wanting strict typings or to iterate over values such as with `Object.keys(TextSize)` they get benefits that aren't available from simply having union types. With that said, the types are still aligned here and we essentially get the best of both worlds. There's a publish coming from FAST with a helper for setting up the types in a more ergonomic fashion. Here's the PR for that for reference: https://github.com/microsoft/fast/pull/6575